### PR TITLE
Bugfix/eicnet 971 make story author not linkable

### PR DESCRIPTION
**Pull request by xwero**
https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-971?focusedCommentId=4948934&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4948934

The other comments in the ticket are not actionable.

### CODE REVIEW

**Review by lramarojaona — CHANGES_REQUESTED** [review]

File: lib/themes/eic_community/includes/preprocess/layout/page.inc [hunk 1]
```diff
@@ -149,7 +149,7 @@ function _preprocess_node_page(&$variables) {
         $author_theme = [
           '#theme' => 'eic_community_author',
           '#name' => $author['name'],
-          '#path' => $author['path'],
+          '#path' => $variables['logged_in'] ? $author['path'] : '',
```
[the remark this answers is not in the document]
Comment (lramarojaona) [reply lib/themes/eic_community/includes/preprocess/layout/page.inc:152]: Or just check the 'access user profiles' permission against current user if the function doesn't fit well in here.

[the remark this answers is not in the document]
Comment (xwero) [reply lib/themes/eic_community/includes/preprocess/layout/page.inc:152]: Bernardo changed the eic_community_get_teaser_user_display function to use the access method of the Url object. So i need to rewrite this.

[the remark this answers is not in the document]
Comment (xwero) [reply lib/themes/eic_community/includes/preprocess/layout/page.inc]: The code is updated now.

[the remark this answers is not in the document]
Comment (lramarojaona) [reply lib/themes/eic_community/includes/preprocess/layout/page.inc]: Cool! :)

**Review by lramarojaona — APPROVED** [review]

**Pull request merged**